guardian_linter_watchdog.py

```
@@ -21,43 +20,54 @@
+                f"Guardian: Found issues in {
+                    file_path.relative_to(PROJECT_ROOT)}")
             if stdout:
                 print("\n--- LINT REPORT ---")
                 print(stdout.decode().strip())
                 print("--- END REPORT ---\n")
             if stderr:
-                logger.error(f"Linter error on {file_path.relative_to(PROJECT_ROOT)}:\n{stderr.decode().strip()}")
+                logger.error(
+                    f"Linter error on {
+                        file_path.relative_to(PROJECT_ROOT)}:\n{
+                        stderr.decode().strip()}")
```

Ruff (0.11.9)

Logging statement uses f-string

(G004)